Makefile

```diff
@@ -97,6 +98,14 @@ package: $(PKG_ROOT)/$(PKG)
 clean-package:
 	rm -rf $(PKG_ROOT)
 
+.PHONY: clean-profile
+clean-package:
```

Comment:
clean-profile?

Reply:
Also, ow do i delete all of the *.txt files generated during profile invocations?